CREATE-NEW-SECRET-SOURCE.md

```diff
@@ -16,7 +16,7 @@ public class Program
             Host.CreateDefaultBuilder(args)
                 .ConfigureSecretStore((context, config, builder) =>
                 {
-                    builder.AddEnvironmentVariableProvider();
+                    builder.AddEnvironmentVariableSecrets();
```

Comment:
We're adding it to the secret store builder so duplication is not needed I think
```suggestion
                    builder.AddEnvironmentVariables();
```